src/Icons/README.md

```diff
@@ -0,0 +1,77 @@
+# Symfony UX Icons
```

Comment:
I do like the README (and sometimes think the symfony ones are a bit.... rough)

As the other packages uses this "standard" (not sure it's the good term), should we writing this content in the same RST file: `doc/index.rst` ?

Reply:
I used the readme to document this feature as it's easy to review on github (and easier for me to write). Before release, we'll switch to rst.

Reply:
This answer makes so much sense I feel stupid haha